app/react-native/src/bin/storybook-start.js

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
 /* eslint-disable no-console */
-
+/* eslint-disable-next-line camelcase */
+import child_process from 'child_process';
```

Comment:
Any reason you don't like camelCase?

Reply:
@sky-franciscogoncalves Do you think you could fix this minor comment?, and we can get this in the next major release!